pkgs/top-level/aliases.nix

```diff
@@ -41,6 +41,7 @@ mapAliases ({
   ag = silver-searcher; # added 2018-04-25
   aircrackng = aircrack-ng; # added 2016-01-14
   alienfx = throw "alienfx has been removed."; # added 2019-12-08
+  aleth = throw "aleth (previously packaged as cpp_ethereum) has been removed; abandoned upstream."; # added 2020-11-30
```

Comment:
Why are you adding this? This package was never in nixpkgs.

Reply:
cpp-ethereum was renamed to aleth quite a while back, we just never renamed it on our end - If someone comes looking for it now, they will likely be looking for aleth